css/css-contain/contain-layout-001.html

```diff
@@ -11,9 +11,14 @@
 <style>
 span {
```

Comment:
This selector should not be `span`, but should only select the first of the two spans. Whether you want to add an id or use `:first-of-type` or whatever is up to you, but it should only select the first span.

Reply:
Done.